components/security/authorization.rst

```diff
@@ -105,10 +105,14 @@ AuthenticatedVoter
 ~~~~~~~~~~~~~~~~~~
 
 The :class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\AuthenticatedVoter`
-voter supports the attributes ``IS_AUTHENTICATED_FULLY``, ``IS_AUTHENTICATED_REMEMBERED``,
-and ``IS_AUTHENTICATED_ANONYMOUSLY`` and grants access based on the current
-level of authentication, i.e. is the user fully authenticated, or only based
-on a "remember-me" cookie, or even authenticated anonymously?::
+voter supports the attributes ``IS_AUTHENTICATED_FULLY``,
+``IS_AUTHENTICATED_REMEMBERED``, ``IS_AUTHENTICATED_ANONYMOUSLY``,
+``IS_ANONYMOUS``, ``IS_REMEMBERED``, ``IS_IMPERSONATED`` and grants access
+based on the current level of authentication, i.e. is the user fully
```

Comment:
I think this sentence is not really accurate anymore, the first three are about `the current level of authentication` indeed, but the new ones are about a `specific level` of authentication.